frontend/src/pages/FavoritePage.tsx

```diff
@@ -0,0 +1,50 @@
+import {Container, Row} from "react-bootstrap";
+import React, {useEffect, useState} from "react";
+import {useNavigate, useParams} from "react-router-dom";
+import {FavoriteNft} from "../model/FavoriteNft";
+import FavoriteGallery from "../components/FavoriteGallery";
+
+
+
+
+type FavoriteProps ={
+    favoriteNfts: FavoriteNft[];
+    getAllFavorite: ()=>void
```

Comment:
getAllFavorite should have a type. Example - getAllFavorite:(here: Here) => void;